wallet/node/Main.hs

```diff
@@ -115,6 +116,8 @@ action (WalletNodeArgs (cArgs@CommonNodeArgs{..}) (wArgs@WalletArgs{..})) =
 main :: IO ()
 main = do
     args <- getWalletNodeOptions
-    CLI.printFlags
-    putText "[Attention] Software is built with wallet part"
-    runProduction $ action args
+    let loggingParams = CLI.loggingParams "wallet" (wnaCommonNodeArgs args)
+    loggerBracket loggingParams . logException "wallet" $ do
```

Comment:
I don't think it's a good idea to use `wallet` logger name here. End-users run `cardano-node` with `log-config-prod.yaml` (from the root of the repository). Only logs sent to `node` logger (and subloggers) will be sent to files. So basically there will be no logs.
I suggest that we leave `node` here (this line and one above) in cardano-sl-1.0 branch to minimize changes (otherwise we'd have to modify log config which would be another moving part).